src/pages/productDetail/ProductDetail.page.jsx

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import Button from "../../components/button/Button.component";
+import GalleryImages from "../../components/galleryImages/GalleryImages.component";
+import Loading from "../../components/loading/Loading.component";
+import Tabs from "../../components/tabs/Tabs.component";
+import Tag from "../../components/tag/Tag.component";
+import { useDetailProduct } from "../../utils/hooks/useDetailProduct";
+import {
+  StyledContainerDetail,
+  StyledImageCart,
+  StyledLabelInput,
+  StyledLabelName,
+  StyledLabelPrice,
+  StyledLabelReturn,
+  StyledLabelSku,
+  StyledWrapperCard,
+  StyledWrapperLabel,
+  StyledWrapperLeft,
+  StyledWrapperPrice,
+  StyledWrapperPriceDetail,
+  StyledWrapperPriceQuantity,
+  StyledWrapperQuantity,
+  StyledWrapperQuantityAll,
+  StyledWrapperRight,
+  StyledLabelReturnMobile,
+  StyledLabelNameDetails,
+  StyledLabelSkuDetails,
+  StyledLabelCategoryDetails,
+  StyledLabelPriceDetails,
+  StyledWrapperPriceDetails,
+  StyledWrapperTags,
+} from "./ProductDetail.styles";
+
+function ProductDetailPage() {
+  const navigate = useNavigate();
```

Comment:
Here we can add the pagination by defining:
-     Max Number of results per Page (e.g. MaxRes)
-     Number of pages that we will have available (e.g. res/MaxRes )
-     Current page value (e.g. use State, or another variable in the URL)
This values could help implement a functionality to view paginated results